src/EA.Weee.Domain.Tests.Unit/Producer/ProducerSubmissionTests.cs

```diff
@@ -338,6 +338,51 @@ public void Producer_SetProducerPartnershipCountry_Returns_RegOfficeOrPPoBCountr
             Assert.Equal("SPAIN", producer.RegOfficeOrPBoBCountry);
         }
 
+        [Fact]
+        public void Producer_SetProducerMemberUpload_HasAnnualCharge_Returns_Yes()
```

Comment:
Create a Producer_SetProducerMemberUpload_HasAnnualCharge_Returns_No test unless its somewhere I cant see...

Reply:
There is one but for SpgInvoiceRunChargeBreakdownTests.Execute_ReturnsProducer_HasAnnualCharge_ReturnsNo() I'll create another one in ProducerSubmissionTests for No flag